frontend/src/container/PlannedDowntime/DropdownWithSubMenu/DropdownWithSubMenu.styles.scss

```diff
@@ -0,0 +1,119 @@
+.options {
+	width: 100%;
+}
+
+.option {
+	padding: 8px 10px;
+	cursor: pointer;
+	overflow: auto;
+}
+
+.option:hover {
+	background-color: var(--bg-slate-200);
+}
+
+.submenu-container {
+	position: absolute;
+	top: 0;
+	right: 50%;
+	z-index: 1;
+	background-color: var(--bg-ink-400);
+	border: 1px solid var(--bg-slate-500);
+	max-height: 300px;
+	overflow-y: auto;
+	width: 160px;
+	display: flex;
+	flex-direction: column;
+	gap: 6px;
+	padding: 12px;
+	border-radius: 4px;
+}
+
+.dropdown-submenu {
+	input[type='text'] {
+		box-sizing: border-box;
+		margin: 0;
+		padding: 4.5px 11px;
+		color: rgba(255, 255, 255, 0.85);
+		font-size: 13px;
+		line-height: 1.6153846153846154;
+		list-style: none;
+		font-family: Inter;
+		position: relative;
+		display: inline-block;
+		min-width: 0;
+		transition: all 0.2s;
+	}
+
+	.ant-popover-inner {
+		padding: 0px;
+	}
+}
+
+.options-container {
+	position: relative;
+	--arrow-x: 175px;
+	--arrow-y: 266px;
+	width: 350px;
+	box-sizing: border-box;
+	margin: 0;
+	padding: 4px;
+	color: var(--bg-vanilla-400);
+	font-size: 12px;
+	line-height: 1.6153846153846154;
+	list-style: none;
+	font-family: Inter;
+	z-index: 1050;
+	overflow: hidden;
+	font-variant: initial;
+	background-color: var(--bg-ink-400);
+	border-radius: 2px;
+	outline: none;
+	box-shadow: 4px 10px 16px 2px rgba(0, 0, 0, 0.3);
+}
+
+.submenu-popover {
+	.ant-popover-inner {
+		padding: 0px;
+	}
+}
+
+.save-option-btn {
+	height: 24px;
+	padding: 0;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	align-self: flex-end;
+	width: 60px;
+}
+
+.submenu-header {
+	color: #52575c;
```

Comment:
do we have variable for the same ?

Reply:
used - 

dark mode - vanilla-200
light mode - slate-200